.github/workflows/ci-test-workflow.yml

```diff
@@ -15,18 +15,18 @@ jobs:
     - name: Run conftest
       uses: stefanprodan/kube-tools@v1
       with:
-        kubectl: 1.21.1
-        kustomize: 3.9.4
-        helmv3: 3.6.3
+        kubectl: 1.23.4
+        kustomize: 4.5.2
+        helmv3: 3.8.1
         command: |
           set -x
           set +e
           kustomize build envs/ci-testing | conftest test --update github.com/instrumenta/policies.git//kubernetes - -o json > result.json || echo "conftest failed!"
           set -e
-          cat result.json #| jq '.[0].failures[].msg' | xargs printf "::warning::%s\n"
+          cat result.json | jq -r '(.[0].failures // []) | "::warning::\(.[].msg)\n"'
     - uses: engineerd/setup-kind@v0.5.0
       with:
-        version: v0.11.1
+        version: v0.12.0
```

Comment:
For k8s-cluster-api-provider, we are using kind-0.14 now (with k8s-1.24) on the management cluster; k8s versions for workload clusters can be freely chosen.
I guess we'll move to kind-0.14 later for this CI testing.

Reply:
Yes. The only question would be whether we should update here, update `generics` and do `gilt overlay` here OR update `generics` and then do `gilt overlay` here.

I'll proceed with the first option, so that kind is updated immediately.